$OpenBSD: patch-Makefile,v 1.5 2016/05/28 13:47:14 sthen Exp $

# XXX N.B. if we get waitid() and want to enable parallel,
# it conflicts with sysutils/parallel

--- Makefile.orig	Sat Apr  2 20:23:51 2016
+++ Makefile	Sat May 28 14:44:56 2016
@@ -1,4 +1,5 @@
-BINS=isutf8 ifdata ifne pee sponge mispipe lckdo parallel errno
+BINS=isutf8 ifdata ifne pee sponge mispipe lckdo errno
+#parallel - wants waitid()
 PERLSCRIPTS=vidir vipe ts combine zrun chronic
 MANS=sponge.1 vidir.1 vipe.1 isutf8.1 ts.1 combine.1 ifdata.1 ifne.1 pee.1 zrun.1 chronic.1 mispipe.1 lckdo.1 parallel.1 errno.1
 CFLAGS?=-O2 -g -Wall
@@ -18,8 +19,8 @@ install:
 	$(INSTALL_BIN) $(BINS) $(DESTDIR)$(PREFIX)/bin
 	install $(PERLSCRIPTS) $(DESTDIR)$(PREFIX)/bin
 	
-	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	install $(MANS) $(DESTDIR)$(PREFIX)/share/man/man1
+	mkdir -p $(DESTDIR)$(PREFIX)/man/man1
+	install $(MANS) $(DESTDIR)$(PREFIX)/man/man1
 
 check: isutf8
 	./check-isutf8
